app/models/file_set.rb

```diff
@@ -4,4 +4,8 @@ class FileSet < Valhalla::Resource
   attribute :id, Valkyrie::Types::ID.optional
   attribute :title, Valkyrie::Types::Set
   attribute :member_ids, Valkyrie::Types::Array
+
+  def thumbnail_id
+    @thumbnail_id ||= Valkyrie.config.metadata_adapter.query_service.find_members(resource: self).find { |x| x.use.include?(Valkyrie::Vocab::PCDMUse.ServiceFile) }.try(:id)
```

Comment:
I agree this is confusing, but what do you mean by image API suffix? I wondered if we just need a derivative_file_metadata method